.byebug_history

```diff
@@ -0,0 +1,50 @@
+continue
+next
+continue
+contineut
+next
+@piece_white
+white_piece
+next
+white_piece
+continue
+next
+continue
+player.pieces
+player
+next
+occupied_space?(destination_row, destination_column)
+destination_column
+destination_row
+continue
+next
+@name
+next
+params
+continue
+next
+params
+@name
+next
+params[:bob]
+continue
+params
+params[:test] = "hello"
+params
+exit
+next
+quit
+next
+quit
+next
+quit
+next
+quit
+next
+quit
+finish
+bt
+next
+pry
+next
+info
```

Comment:
can you remove this file? doesn't need to be commited. Can be added to `.gitignore`

more info: https://git-scm.com/docs/gitignore